spotipyn.py

```diff
@@ -8,4 +8,5 @@
 spotipyn = create_app()
 
 if __name__ == "__main__":
+    spotipyn.debug = True
```

Comment:
Please remove this and use this line locally. We're not setting debug mode to `True` on the production branch.